build/build.sh

```diff
@@ -0,0 +1,24 @@
+export http_proxy_host=$(echo $http_proxy | awk -F':' {'print $2'} | tr -d '/')
+export http_proxy_port=$(echo $http_proxy | awk -F':' {'print $3'} | tr -d '/')
+
+export https_proxy_host=$(echo $https_proxy | awk -F':' {'print $2'} | tr -d '/')
+export https_proxy_port=$(echo $https_proxy | awk -F':' {'print $3'} | tr -d '/')
+
+export _JAVA_OPTIONS="-Dhttp.proxyHost=$http_proxy_host -Dhttp.proxyPort=$http_proxy_port -Dhttps.proxyHost=$https_proxy_host -Dhttps.proxyPort=$https_proxy_port"
+export JAVA_HOME=/usr/lib/jvm/java-11-openjdk-amd64
+
+REMOTE_URL=https://github.com/secure-device-onboard/rendezvous-service.git
+REMOTE_BRANCH=master
+
+if [ "$use_remote" = "1" ];
```

Comment:
The indent of 'then' should be at the same level as 'if'. Also use 2-spaces for single indent level.
if [ condition ]; then
. . statement
else
. . statement
fi

or 
if [ condition ]; 
then
. . statement
else
. . statement
fi

Reply:
Changed the indentation